application/Profile/DBProfile.py

```diff
@@ -273,6 +309,8 @@ def add_session(self, host, username, conn=None):
                 session.private_key = f.encrypt(clear_private_key).decode('ascii')
 
             self.save_profile()
+
+            logger.info("Successfully added a new session: session_id = %s", session.id)
```

Comment:
for spacing consistency with above
```suggestion
            logger.info("Successfully added a new session: session_id=%s", session.id)
```